### ISSUE TITLE
Property deleter not being ignored

### ISSUE THREAD

**Issue by ts-mk** [econchick/interrogate#126]
### Environment

- `interrogate` version(s) (`interrogate --version`): 1.5.0
- Operating System(s): macOS
- Python version(s): 3.9.10

### Description of the bug

With `ignore-property-decorators` set to `true`, a property deleter is not being ignored.

### What you expected to happen

A property deleter should be ignored the same way a property getter and setter is.

### How to reproduce (as minimally and precisely as possible)

```python
class C:
    @property
    def x(self):
        return self._x

    @x.setter
    def x(self, value):
        self._x = value

    @x.deleter
    def x(self):
        del self._x
```

**Comment by econchick** [comment]
Just wanted to let you know that I have a fix, and will be released in the next release (`22.0` - switching to calendar versioning), hopefully by the end of the month - I'll update the issue when it's out. 

Thank you for your patience!

**Comment by econchick** [comment]
Released! feel free to re-open if this is still an issue.
